x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -267,10 +281,29 @@ export class TaskManagerPlugin
       requestTimeouts: this.config.request_timeouts,
     });
 
+    const defaultCapacity = getDefaultCapacity({
+      claimStrategy: this.config?.claim_strategy,
+      heapSizeLimit: this.heapSizeLimit,
+      isCloud: cloud?.isCloudEnabled ?? false,
+      isServerless: !!serverless,
+      isBackgroundTaskNodeOnly: this.isNodeBackgroundTasksOnly(),
+    });
+
+    this.logger.info(
```

Comment:
@mikecote do you think it's ok to keep this at info level? it should only log one time on startup.

Reply:
I think for now it's ok, we're making frequent enough changes that will be useful to know this information while debugging. We can keep it for now.